src/proxyfmu/fmi/fmi2/fmi2_slave.cpp

```diff
@@ -29,6 +29,7 @@ fmi2_slave::fmi2_slave(
     , md_(std::move(md))
     , tmpDir_(std::move(tmpDir))
     , handle_(fmi2_import_parse_xml(ctx->ctx_, tmpDir->path().string().c_str(), nullptr))
+    , instanceName_(instanceName)
```

Comment:
Caught a potential bug. I see `instanceName_` was not used before, so not an issue so far.

Reply:
Nevermind, the field was added in this PR